### ISSUE TITLE
web mode fails at runtime, no usable features

### ISSUE THREAD

**Issue by covercash2** [fxbois/web-mode#852]
when I start web mode it gives the error: `Wrong number of arguments: (2 . 2), 1`

I'm new to emacs so I'm not sure how to interpret this message. I found some leads that it might be symlinks or daemon mode, but I cp'd the directory and tried it GUI mode (no daemon) with the same result. 

macOS 10.11.6
emacs 25.1

[dotfiles](https://github.com/covercash2/dotfiles) (it's a mess in there, but init.el is easy to find)

thanks in advance

**Comment by fxbois** [comment]
@covercash2 could you try without daemon mode and with a very "light" .dotemacs ?

**Comment by fxbois** [comment]
@covercash2 please reopen if needed

**Comment by covercash2** [comment]
sorry for the delay. 

[here's](http://pastebin.com/czjRcsSy) my minimal init.el

I took the opportunity to clean up the exec-path. I also removed the symlink and copied the files over and ran in non-daemon mode. same error.

**Comment by fxbois** [comment]
@covercash2 could you switch to the ```*Messages*``` buffer and copy/paste the related message ?

**Comment by covercash2** [comment]
right, so I figured out, to my embarrassment, that the web-mode functions were working. I hadn't even thought to try since syntax highlighting was off and there was an error message. I did a web-mode-reload and everything started working without error. syntax highlighting, element highlights, go template engine, etc.

earlier today I tracked a lead on abbrev-table-get with some long confusing args that basically lead nowhere. I tried to get into the emacs debuggers but was pretty confounded. 

I'm just new at emacs. I'd be happy to help you track this down, but as far as I'm concerned doing a reload after startup isn't the end of the world.
